src/Core/AndroidNative/build.gradle

```diff
@@ -0,0 +1,26 @@
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
+buildscript {
+    repositories {
+        google()
+        mavenCentral()
+    }
+    dependencies {
+        classpath "com.android.tools.build:gradle:7.0.3"
+    }
+}
+
+task clean(type: Delete) {
+    delete rootProject.buildDir
+}
+
+task createAar(type: Copy){
+    dependsOn ':maui:clean'
+    dependsOn ':maui:assembleRelease'
+
+    tasks.getByPath(':maui:assembleRelease').mustRunAfter(tasks.getByPath(':maui:clean'))
+
+    from('maui/build/outputs/aar/maui-release.aar')
+    into('../src/')
+    include('maui-release.aar')
+    rename('maui-release.aar', 'maui.aar')
+}
```

Comment:
I might move this call into MSBuild, so you'd just build the `.csproj` and don't have to know about this.

I also need to make the C# bindings for `maui.aar` in a better C# namespace, and make all members `internal`.